app/core/settings.py

```diff
@@ -88,6 +92,55 @@
     'DATETIME_FORMAT': "%Y-%m-%dT%H:%M:%SZ",
 }
 
+LOG_ATTRS = ['asctime', 'created', 'levelname', 'message',
+             'filename', 'funcName', 'lineno', 'module', 'pathname']
+LOG_FORMAT_STR = ' '.join(['%(' + v + ')s' for v in LOG_ATTRS])
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'json': {
+            '()': jsonlogger.JsonFormatter,
+            'format': LOG_FORMAT_STR,
+        },
+        'verbose': {
+            'format': "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s",
+            'datefmt': "%d/%b/%Y %H:%M:%S"
+        },
+        'simple': {
+            'format': '%(levelname)s %(message)s'
+        },
+    },
+    'handlers': {
+        'file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': os.path.join(DEFAULT_LOG_DIR, '%s.json.log' % PROJECT_NAME),
+            'formatter': 'json'
+        },
+        'console': {
+            'class': 'logging.StreamHandler'
+        },
+    },
+    'loggers': {
+        '': {
```

Comment:
assuming this is the root logger, it should go to a file too to catch logs from libraries like `boto`? Would remove the need for the separate loggers for `core` and `digests`?

Reply:
I would suggest we keep them separate then if you want all going to the file so that by default we only go as far down as `INFO` and the custom modules can remain at `DEBUG`.

Reply:
Good for default at `INFO` and custom modules at `DEBUG`